src/main/scala/com/fortysevendeg/android/scaladays/modules/net/client/messages/Messages.scala

```diff
@@ -0,0 +1,31 @@
+package com.fortysevendeg.android.scaladays.modules.net.client.messages
+
+import play.api.libs.json.Reads
+
+case class ServiceClientStringRequest(
+  path: String,
+  headers: Seq[(String, String)] = Seq.empty,
+  emptyResponse: Boolean = false)
+
+case class ServiceClientStringResponse(statusCode: Int, data: Option[String])
+
+case class ServiceClientRequest[Res](
+  path: String,
+  headers: Seq[(String, String)] = Seq.empty,
+  reads: Option[Reads[Res]] = None,
+  emptyResponse: Boolean = false)
+
+case class ServiceClientWithBodyRequest[Res](
+  path: String,
+  headers: Seq[(String, String)] = Seq.empty,
+  body: Map[String, String],
```

Comment:
We should add a default value for `body` param or move `header` param bellow `body`